tests/test_items.py

```diff
@@ -65,6 +82,25 @@
     ),
     "variants": [ProductVariant()],
 }
+_PRODUCT_LIST_MIN_KWARGS = {
+    "url": "https://example.com/?product=product22",
```

Comment:
a nitpick: something which looks like a product list url could be a better example, e.g. example.com/watches/swiss (to match categoryName)